addons/hr_expense/models/res_company.py

```diff
@@ -6,6 +6,12 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
+    expense_category_id = fields.Many2one(
+        "product.product",
+        string="Default Expense Category",
+        check_company=True,
+        domain="[('can_be_expensed', '=', True), '|', ('company_id', '=', False), ('company_id', '=', company_id)]",
+    )
```

Comment:
field name is confusing: sounds like a product.category

You should rename that into expense_product_id

Reply:
okok